Assets/Scripts/Internal/DaggerfallInterior.cs

```diff
@@ -471,6 +473,17 @@ private void AddModels(PlayerGPS.DiscoveredBuilding buildingData, bool isAutomap
                         dfMesh.SetClimate(climateBase, climateSeason, WindowStyle.Disabled);
                     }
                 }
+                else
+                {
+                    // For a custom model initialise any custom doors
+                    List<StaticDoor> customStaticDoors = CustomDoor.InitDoorsInterior(modelGO, staticDoors, entryDoor.blockIndex, entryDoor.recordIndex, modelMatrix, out disableClassicDoors);
+                    doors.AddRange(customStaticDoors);
+                }
+
+                // Add any DF model static doors unless suppressed
+                if (modelData.Doors != null && !disableClassicDoors)
+                    doors.AddRange(GameObjectHelper.GetStaticDoors(ref modelData, entryDoor.blockIndex, entryDoor.recordIndex, modelMatrix));
```

Comment:
Do we have to call `GameObjectHelper.GetStaticDoors(ref modelData, entryDoor.blockIndex, entryDoor.recordIndex, modelMatrix)` a second time here, or would reusing `staticDoors` work?

Reply:
Yeah it would, I'll push a change when I get the chance to test it.

Reply:
Tested and committed.